mobile/library/common/jni/BUILD

```diff
@@ -34,38 +55,67 @@ cc_library(
     deps = [
         ":android_network_utility_lib",
         ":jni_utility_lib",
-        ":ndk_jni_support",
         "//library/common:envoy_main_interface_lib",
         "//library/common/api:c_types",
+        "//library/common/extensions/cert_validator/platform_bridge:c_types_lib",
+        "//library/common/extensions/key_value/platform:config",
         "//library/common/types:managed_types_lib",
     ],
     # We need this to ensure that we link this into the .so even though there are no code references.
     alwayslink = True,
 )
 
+# Cert verification related functions which call into AndroidNetworkLibrary.
 cc_library(
-    name = "envoy_jni_lib",
+    name = "android_network_utility_lib",
+    srcs = [
+        "android_network_utility.cc",
```

Comment:
should we keep the `android_network_utility_lib` target at the bottom where it was before, so it makes the diff between the two easier to follow?  i think there were no diffs but not 100% sure

Reply:
I did consider that, and I'm happy to move it back. you're right that there are no changes. I chose to move it here so that the BUILD file could be read in-order, if that makes sense. In other words, I wanted to make sure we didn't depend on target until we'd seen them, otherwise it's pretty hard to keep the right stuff in my mind. But happy to move it back if you prefer?

Reply:
I like your idea of keeping dependencies above their dependents, that's how I like to structure BUILD files too, so lets keep it as is.  Just wanted to make sure I didn't miss anything in the diffs but since you confirmed it, I'm good :)